operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller_test.go

```diff
@@ -57,11 +56,18 @@ var (
 		string(name.PilotComponentName):     healthyVersionStatus,
 		string(name.TelemetryComponentName): healthyVersionStatus,
 		string(name.PolicyComponentName):    healthyVersionStatus,
-		string(name.CitadelComponentName):   healthyVersionStatus,
 		string(name.IngressComponentName):   healthyVersionStatus,
 		string(name.EgressComponentName):    healthyVersionStatus,
 		string(name.AddonComponentName):     healthyVersionStatus,
 	}
+	sdsStatus = map[string]*v1alpha1.InstallStatus_VersionStatus{
```

Comment:
Why did this add the `sdsStatus` back without using it in the tests? Is this a merge mishap?

Reply:
Looks like a merge artifact; this PR was older than the sds profile removal. Filed #20444 to remove

Reply:
We'd better add test case for `SDS` profile.